src/ansys/geometry/core/designer/design.py

```diff
@@ -292,6 +293,180 @@ def download(
             f"Design is successfully downloaded at location {file_location}."
         )
 
+    def export_to_scdocx(self, location: Union[Path, str] = None) -> str:
+        """
+        Export the design to an scdocx file.
+
+        Parameters
+        ----------
+        location : Union[~pathlib.Path, str], optional
+            Location on disk to save the file to. If None, the file will be saved
+            in the current working directory.
+
+        Returns
+        -------
+        str
+            The path to the saved file.
+        """
+        # Define the file location
+        file_location = (Path(location) if location else Path.cwd()) / f"{self.name}.scdocx"
+
+        # Export the design to an scdocx file
+        self.download(file_location, DesignFileFormat.SCDOCX)
+
+        # Return the file location
+        return file_location
+
+    def export_to_parasolid_text(self, location: Union[Path, str] = None) -> str:
+        """
+        Export the design to a Parasolid text file.
+
+        Parameters
+        ----------
+        location : Union[~pathlib.Path, str], optional
+            Location on disk to save the file to. If None, the file will be saved
+            in the current working directory.
+
+        Returns
+        -------
+        str
+            The path to the saved file.
+        """
+        # Determine the extension based on the backend type
+        ext = "x_t" if self._grpc_client.backend_type == BackendType.LINUX_SERVICE else "xmt_txt"
+
+        # Define the file location
+        file_location = (Path(location) if location else Path.cwd()) / f"{self.name}.{ext}"
+
+        # Export the design to a Parasolid text file
+        self.download(file_location, DesignFileFormat.PARASOLID_TEXT)
+
+        # Return the file location
+        return file_location
+
+    def export_to_parasolid_bin(self, location: Union[Path, str] = None) -> str:
+        """
+        Export the design to a Parasolid binary file.
+
+        Parameters
+        ----------
+        location : Union[~pathlib.Path, str], optional
+            Location on disk to save the file to. If None, the file will be saved
+            in the current working directory.
+
+        Returns
+        -------
+        str
+            The path to the saved file.
+        """
+        # Determine the extension based on the backend type
+        ext = "x_b" if self._grpc_client.backend_type == BackendType.LINUX_SERVICE else "xmt_bin"
+
+        # Define the file location
+        file_location = (Path(location) if location else Path.cwd()) / f"{self.name}.{ext}"
```

Comment:
Minor, but I would implement this method instead of these lines in export methods, up to you.

```python
def build_file_location(self, ext: str, location: Union[Path, str] = None) -> Path:
    return (Path(location) if location else Path.cwd()) / f"{self.name}.{ext}"

```

Reply:
I like that. I'll just keep the method private.